google-cloud-functions/README.md

```diff
@@ -31,6 +39,42 @@ To access this integration, [connect to Google Cloud Platform](https://docs.sign
 
 None
 
+### Wrappers
+
+#### SignalFx Wrapper vs CloudWatch monitoring
```

Comment:
Google's cloud monitoring service is called Google Cloud Monitoring (rebranded from Google StackDriver, but I don't think we have changed the branding in our own documentation yet).
CloudWatch is specific to Amazon